tests/testevent.c

```diff
@@ -165,6 +165,107 @@ static LWES_BYTE ref_bytes_db[188] = {
   0xff,0xff,0xff,0x0b,0x61,0x4d,0x65,0x74,0x61,0x53,0x74,0x72,0x69,0x6e,0x67,
   0x05,0x00,0x05,0x68,0x65,0x6c,0x6c,0x6f};
 
+static int
+lwes_event_to_stream
+  (struct lwes_event *event,
+   FILE *stream)
+{
+  struct lwes_event_attribute *tmp;
+  struct lwes_hash_enumeration e;
+
+  fprintf (stream, "%s", event->eventName);
+  fprintf (stream,"[");
+  fflush (stream);
+  lwes_typed_value_to_stream (LWES_TYPE_U_INT_16, (void*)&event->number_of_attributes, stream);
+  fprintf (stream,"]");
+  fflush (stream);
+  fprintf (stream,"\n");
+  fflush (stream);
+  fprintf (stream,"{\n");
+  fflush (stream);
+
+  if (lwes_hash_keys (event->attributes, &e))
+    {
+      while (lwes_hash_enumeration_has_more_elements (&e))
+        {
+          LWES_SHORT_STRING tmpAttrName =
+            lwes_hash_enumeration_next_element (&e);
+
+          tmp =
+            (struct lwes_event_attribute *)lwes_hash_get (event->attributes,
+                                                          tmpAttrName);
+
+          fprintf (stream,"\t");
+          fflush (stream);
+          fprintf (stream, "%s", tmpAttrName);
+          fflush (stream);
+          fprintf (stream," = ");
+          fflush (stream);
+          lwes_event_attribute_to_stream (tmp, stream);
+          fflush (stream);
+          fprintf (stream,";\n");
+          fflush (stream);
+        }
+    }
+  fprintf (stream,"}\n");
+  fflush (stream);
+  return 0;
+}
```

Comment:
This again looks like the same function from the command line programs.  I think we need to put it into the library somewhere?